server/app/routes/v1/grids_api.rb

```diff
@@ -127,8 +127,16 @@ class GridsApi < Roda
             end
             unless r['services'].nil?
               services = r['services'].split(',').map do |service|
-                @grid.grid_services.find_by(name: service).try(:id)
-              end.delete_if{|s| s.nil?}
+                if service.include?('/')
+                  stack_name, service_name = service.split('/', 3).last(2)
+                  stack_id = @grid.stacks.where(name: stack_name).first.try(:id)
+                  if stack_id && service_name
+                    @grid.grid_services.where(stack_id: stack_id, name: service_name).first.try(:id)
+                  end
```

Comment:
I wouldn't say it's a good practice, but okay, I'll make a separate issue for it:  #2714